packages/@okta/vuepress-theme-prose/assets/css/abstracts/_variables.scss

```diff
@@ -11,13 +11,13 @@ $asset-path: '/assets';
 //-----------------------------------------------------------------------
 // Typography
 //-----------------------------------------------------------------------
-$font-family: 'proxima-nova', Helvetica, Arial, sans-serif;
+$font-family: 'ABC Whyte', 'proxima-nova', Helvetica, Arial, sans-serif;
```

Comment:
Should we replace `proxima-nova` rather than move it in the list? I don't think there's a case where we want `proxima-nova` as a fallback